samples/apps/copilot-chat-app/webapi/CopilotChat/Storage/ChatParticipantRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SemanticKernel.Service.CopilotChat.Models;
+
+namespace SemanticKernel.Service.CopilotChat.Storage;
+
+/// <summary>
+/// A repository for chat sessions.
+/// </summary>
+public class ChatParticipantRepository : Repository<ChatParticipant>
+{
+    /// <summary>
+    /// Initializes a new instance of the ChatParticipantRepository class.
+    /// </summary>
+    /// <param name="storageContext">The storage context.</param>
+    public ChatParticipantRepository(IStorageContext<ChatParticipant> storageContext)
+        : base(storageContext)
+    {
+    }
+
+    /// <summary>
+    /// Finds chat participants by user id.
+    /// A user can be part of multiple chats, thus a user can have multiple chat participants.
+    /// </summary>
+    /// <param name="userId">The user id.</param>
+    /// <returns>A list of chat participants of the same user id in different chat sessions.</returns>
+    public Task<IEnumerable<ChatParticipant>> FindByUserIdAsync(string userId)
```

Comment:
Consider allowing empty return sets to be valid here. Or a TryFind pattern..

Reply:
Same for `FindByChatIdAsync`

Reply:
I believe empty sets are valid return values already.